filters/security/filter_18_PhishingArmy/configuration.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Phishing Army",
+    "description": "Generated every 6 hours from PhishTank, OpenPhish, Cert.pl, PhishFindR, Urlscan.io and Phishunt.io reports.",
```

Comment:
Hello @drego85
What description do you want us to use?

Reply:
Hi, I prefer a more general description because the sources change!

So, I propose: "The Blocklist to filter Phishing domains!"

Thanks

Reply:
Changed, thank you.